Automattic/newspack-popups
# feat: support RAS default prompts wizard UI

### All Submissions:






### Changes proposed in this Pull Request:



Supports new RAS-related setup UI in https://github.com/Automattic/newspack-plugin/pull/2399. The main changes:

- Updates the prompt schema to support additional user input field data types (array and integer in addition to string)
- Implements logic to store user inputs by field name and populate preset config placeholders with matching stored inputs when fetching the preset configs
- Implements logic to build dynamic prompt previews from preset config + user input, without having to create prompt posts

### How to test the changes in this Pull Request:

Follow testing instructions in https://github.com/Automattic/newspack-plugin/pull/2399. The plugin should function as usual and all tests should continue passing even without the changes in https://github.com/Automattic/newspack-plugin/pull/2399.

### Other information:
